main.md

```diff
@@ -875,24 +875,29 @@ prolonged deployments of protected resources with mixed token type support.
 
 # Authorization Server-Provided Nonce {#ASNonce}
 
-Including a nonce value contributed by the authorization server in the DPoP proof
-MAY be used by authorization servers to limit the lifetime of DPoP proofs.
-The server is in control of when to require the use of a new nonce value
-in subsequent DPoP proofs.
-
-Without employing such a mechanism, a malicious party controlling the client
+A malicious party controlling the client
```

Comment:
"Without employing such a mechanism" contains important context that is lost in the replacement text.